swift/extractor/visitors/ExprVisitor.cpp

```diff
@@ -667,5 +667,11 @@ void ExprVisitor::emitLookupExpr(const swift::LookupExpr* expr, TrapLabel<Lookup
     dispatcher_.emit(LookupExprMembersTrap{label, declLabel});
   }
 }
+codeql::UnresolvedPatternExpr ExprVisitor::translateUnresolvedPatternExpr(
```

Comment:
No empty line after the previous function?

Reply:
My IDE insists on that when generating the definition. I hoped there would be a clang-format option for this, and there is (`SeparateDefinitionBlocks`), but unfortunately only from `clang-format` version 14, while we officially use version 13 for now... I will probably install version 14 locally, if there is any formatting difference it will be caught by the pre-commit hook.

Reply:
I don't understand this, because it is and was correct higher up in this file. What's different?

Reply:
The difference is that I was correcting it by hand and forgot it here :sweat_smile:

But now I installed `clang-format-14` and added the configuration to a git excluded `.clangformat`, so now this will be consistent (at least from my side). We can add the `SeparateDefinitionBlocks: Always` option to our `.clangformat` files if and when we decide to move to `clang-format` 14